test/functional/fortran/test_fortran_spg_get_spacegroup_type.F90

```diff
@@ -0,0 +1,67 @@
+module test_spg_get_spacegroup_type
+    use spglib_f08, only: spg_get_spacegroup_type, SpglibSpacegroupType
+    use test_utils
+    use C_interface_module
+    use, intrinsic :: iso_fortran_env
+    implicit none
+contains
+    function test_fortran_spg_get_spacegroup_type(argc, argv) bind(C) result(ret)
+        use, intrinsic :: iso_c_binding
+        integer(c_int), value, intent(in) :: argc
+        type(c_ptr), intent(in) :: argv(argc)
+        integer(c_int) :: ret
+
+        integer :: len
+        character(len=:), allocatable :: str
+
+        if (argc < 2) then
+            write (output_unit, *) "test_dataset_rutile112"
```

Comment:
Same, wrong name input here

Reply:
Thanks for pointing out it. I fixed it as well.